packages/cyberstorm/src/components/Dialog/Dialog.module.css

```diff
@@ -18,8 +17,10 @@
   border: var(--border);
   border-radius: var(--border-radius--8);
   background-color: var(--color-surface--2);
-  box-shadow: var(--box-shadow-default);
-  transform: translate(calc(var(--left) * -1), calc(var(--top) * -1));
+
+  box-shadow: hsl(206deg 22% 7% / 0.35) 0 10px 38px -10px, hsl(206deg 22% 7% / 0.2) 0 10px 20px -15px;
+  transform: translate(-50%, -50%);
```

Comment:
A bit offtopic but is this still the preferred way of centering stuff? I would have thought there's something better by now. @gkurck

Reply:
Yes, flex is preferred here, no need to take the element out of the document flow.